lib/build.js

```diff
@@ -548,7 +564,7 @@ class BuildModel extends BaseModel {
                         jobId: job.id,
                         startTime: this.startTime,
                         annotations: job.permutations[0].annotations,
-                        buildStatus: this.status,
```

Comment:
why is config passed to `executor.startTimer` being modified ?

Reply:
Keeping `buildStatus` in `config`  consistent throughout the system. Should executor queue not maintaining a log of various commit status, this can be removed.

Reply:
scm status should only be passed to scm functions `updateCommitStatus` I don't get why it's beeing passed to executor.startTimer